lib/league_statistics.rb

```diff
@@ -1,48 +1,16 @@
 module LeagueStatistics
 
-# require_relative "./stat_tracker"
-# require "csv"
-#
-# class LeagueStatistics < StatTracker
-
   def count_of_teams
     @teams.count
-    # total_games = CSV.read(@teams, :headers=>true)
-    # total_games.count
   end
 
   def team_name(id)
```

Comment:
Cleaned up code to no longer load csv file every time a method is called. Rather a method is called on an already instantiated collection.